change/@fluentui-web-components-f04748f8-f9ea-46fa-a74b-b301279eb3c3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31302&commitId=85a963fc0d57b20ecea36d17fac77b74d92df8a4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Badge</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge.Appearance - Dark Mode.normal.chromium.png|15| Changed|

</details>


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps.normal.chromium.png|1| Changed|

</details>


  <details><summary><b>Text</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.normal.chromium.png|995| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

Reply:
I don't think these are related to changes. Due to Shadow DOM encapsulation I wouldn't expect other components to be effected by these changes.

- [x] Hexes for this branch against what's in the production Storybook ([Badge w/ theme set to Teams Dark](https://web-components.fluentui.dev/?path=/docs/components-badge-badge--badge#appearance)) for the background of the "tint" variant is the same: `#2f2f4a`
- [x] Slider change appears to be compression noise
- [x] Light theme Text Blocks appear the same on localhost as on prod. 

I've marked these as noise. But let me know if I did something wrong, this is my first PR.

Reply:
Just noise :)